FilmsApp/View/ListController/ListViewController.swift

```diff
@@ -3,11 +3,14 @@ import UIKit
 
 class ListViewController: UIViewController {
     var myTableView = UITableView()
```

Comment:
Use initialization block, the same that was used when initializing collectionView in CollectionListCell